src/App.tsx

```diff
@@ -1,26 +1,46 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
```

Comment:
according to the style guide, import styles should be at the bottom of the imports list